test/unit-test/test_btc_sign.c

```diff
@@ -731,25 +762,45 @@ static void _sign(const _modification_t* mod)
     assert_int_equal(APP_BTC_SIGN_OK, app_btc_sign_input(&inputs[1], &next));
     assert_int_equal(next.type, BTCSignNextResponse_Type_DONE);
     assert_true(next.has_signature);
-    if (mod->is_valid) {
-        const uint8_t expected_signature[64] =
-            "\x95\x09\x43\x09\xa2\xd2\x77\xd3\xa6\x8d\xde\xd3\x3d\x50\xa7\x47\xf2\xee\xfb\x3f\x54"
-            "\x8a\x92\x45\x15\xdb\x62\xbe\x06\xa1\xae\xa4\x56\x92\x91\xe5\x2e\x6f\xea\x95\xf8\xb6"
-            "\x75\x23\xb1\x9b\x35\x9a\x84\x85\xd8\xaa\x3c\xa0\x2d\xb3\x74\x70\x01\x0b\x19\x9b\x0c"
-            "\xe3";
-        assert_memory_equal(next.signature, expected_signature, sizeof(next.signature));
+    if (mod->check_sigs) {
+        switch (mod->script_type) {
+        case BTCScriptConfig_SimpleType_P2WPKH: {
+            const uint8_t expected_signature[64] =
+                "\x95\x09\x43\x09\xa2\xd2\x77\xd3\xa6\x8d\xde\xd3\x3d\x50\xa7\x47\xf2\xee\xfb\x3f"
```

Comment:
The autoformatter kind of failed here? I guess if you concatenate everything into one string and then do clang-format it will look better.

Reply:
Fixed.

Reply:
It doesn't look fixed, but it is only test code so no need to fix.

Reply:
Strange, maybe I fixed another instance of it. Anyway, fixed in https://github.com/digitalbitbox/bitbox02-firmware/pull/348